app/build.gradle

```diff
@@ -39,6 +40,9 @@ android {
     buildFeatures {
         buildConfig = true
     }
+    kotlinOptions {
+        jvmTarget = '1.8'
```

Comment:
@tuancoltech Can we use Java 17 instead?

Reply:
@jo-elimu Sure. Done.